src/App.tsx

```diff
@@ -1,14 +1,17 @@
-import React from "react";
 import "./App.css";
 
 import { taskList } from "./taskList";
 
 function TaskShow() {
   return (
     <div>
+      <div className="title">
+        <h2>My Fancy TODO App</h2>
+      </div>
```

Comment:
this is one way to do it, but since the container doesn't do anything much, we should not leave it here and just use the `h2` instead